float64 finger_distance
---
bool success